### DISCUSSION TITLE
NetBox backups

### DISCUSSION THREAD

**Discussion by bclanton40** [netbox-community/netbox#11524] [General]
Wanted to see what people are doing in regard to backups for their Netbox instance.  Other than backing up the PSQL file, what other items are they backing up should their Netbox server completely fail.

**Comment by mtinberg** [comment]
Uploaded media if you attach images to things, the configuration.py, custom reports/scripts and the install instructions, in my case I'm using the lae.netbox Ansible playbook, everything else you can rebuild and isn't holding (much) state.  You could use pip freeze (I think) to record exactly how the virtualenv was built to be able to restore it to exactly the same state but if the install is within the last 6mo-1y of current that probably won't be a problem, installing the current version of dependencies should be fine.


—

Mark Tinberg ***@***.***>

Division of Information Technology-Network Services

University of Wisconsin-Madison


________________________________
From: bclanton40 ***@***.***>
Sent: Tuesday, January 17, 2023 7:38 AM
To: netbox-community/netbox ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [netbox-community/netbox] NetBox backups (Discussion #11524)


Wanted to see what people are doing in regard to backups for their Netbox instance. Other than backing up the PSQL file, what other items are they backing up should their Netbox server completely fail.
